src/Sylius/Component/Core/Handler/CodeChangeHandlerInterface.php

```diff
@@ -9,14 +9,14 @@
  * file that was distributed with this source code.
  */
 
-namespace Sylius\Component\Core\Locale\Handler;
+namespace Sylius\Component\Core\Handler;
 
 use Sylius\Component\Core\Exception\HandleException;
 
 /**
  * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
  */
-interface LocaleChangeHandlerInterface
+interface CodeChangeHandlerInterface
```

Comment:
Imo locale and currency are two separate concepts and should have two separate interfaces, anyway imo this interface is too generic. WDYT?